core/src/com/team3gdx/game/station/StationManager.java

```diff
@@ -88,6 +88,8 @@ public void checkInteractedTile(String type, Vector2 pos) {
 		case "Onions":
 			takeIngredientStation(pos, Ingredients.onion);
 			break;
+		case "Flour":
+			takeIngredientStation(pos,Ingredients.flour); // added flour ingredient
```

Comment:
Comment unnecessary, that sort of message goes in a commit message or PR description.

Reply:
Unnecessary comment still present